### DISCUSSION TITLE
Exercise 9 cannot run

### DISCUSSION THREAD

**Discussion by Chengwenju** [iiasa/CWatM#48] [Q&A] [answered]
Hello, everyone

I am doing the exercise 9, but the "runCalibration_fast_2.bat" is not successfully run, the error message is “No simulated streamflow found. Probably the model failed to start? Check the log files of the run!”. I tried for a long time, but I don't find the solution to solve it. Who can give me help?

![ERROR](https://user-images.githubusercontent.com/97773815/196176027-aba41f02-0e98-4dd6-a3b9-026d8f0d246e.JPG)
![ERROR1](https://user-images.githubusercontent.com/97773815/196176385-612b62e0-7f21-4304-9b11-165766a13b54.JPG)

**Comment by dof1985** [comment, marked as answer]
Hi Chengwenju

Just some quick lines of thinking.

1. Did you check all log files? are they all without errors? 
2. It is also worthwhile to check that you have updated the models' path in 'runpy.bat'
3. Please check again your settings file - you should set the outputs as follows while commenting out all other outputs from the model.
OUT_Dir = %run_rand_id
OUT_TSS_Daily = discharge
OUT_TSS_MonthAvg = discharge

**Reply by Chengwenju** [reply]
Thanks, dof1985. I have solved this question, the nc data "rhine_19891231.nc" in tutorial exercise9 (../CwatM_model/CWatM-version1.06/tutorial/CWATM_exercise9) is wrong. Replaced it with the correct data to run.
